compositions/upbound-aws-provider/aurora/definition.yaml

```diff
@@ -0,0 +1,187 @@
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+  name: xrelationaldatabases.db.awsblueprint.io
+spec:
+  group: db.awsblueprint.io
+  names:
+    kind: XRelationalDatabase
+    plural: xrelationaldatabases
+  claimNames:
+    kind: RelationalDatabase
+    plural: relationaldatabases
+  connectionSecretKeys:
+    - subnetGroupArn
+    - subnetGroupName
+    - securityGroupId
+    - securityGroupArn
+    - securityGroupName
+    - clusterParameterGroupName
+    - clusterParameterGroupArn
+    - clusterIdentifier
+    - clusterArn
+    - clusterReaderEndpoint
+    - clusterEndpoint
+    - clusterMasterSecretArn 
+    - proxyArn
+    - proxyEndpoint
+    - defaultProxyTgName
+    - targetGroupName 
```

Comment:
Per our discussion we only need proxyEndpoint and we should inject that in a configmap in the same namespace we deploy the Claim, example [here](https://github.com/awslabs/crossplane-on-eks/blob/e3893764d305aaa583e7618282668239d039afa6/compositions/upbound-aws-provider/s3/general-purpose.yaml#L101-L126).

Reply:
will update accordingly